### PR TITLE
Add test for soft dirty bits stability

### PR DESCRIPTION
**Pull request by harish-24**
Patch adds test for soft dirty bits stability

Signed-off-by: Harish <harish@linux.ibm.com>

### CODE REVIEW

**Comment by harish-24** [comment]
```
JOB ID     : 894406495c7c4e187acdc484fb24fbb0853b951f
JOB LOG    : /home/harish/avocado-fvt-wrapper/results/job-2021-02-23T01.44-8944064/job.log
 (1/1) memory/soft_dirty.py:SoftDirtyBits.test: PASS (1.02 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/harish/avocado-fvt-wrapper/results/job-2021-02-23T01.44-8944064/results.html
JOB TIME   : 1.73 s
```

File: memory/soft_dirty.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        self.src_file = self.fetch_asset("test_sd_stability.c",
+                                         locations=locations, expire='7d')
+
+    def test(self):
+        '''
+        Run soft dirty bits test
+        '''
+        os.chdir(self.workdir)
+        process.run("gcc %s -o test_sd_stable" %
+                    self.src_file, shell=True, ignore_status=True)
```
Comment (PraveenPenguin) [inline memory/soft_dirty.py:58]: i feel we need to check if compilation failed as next step will not make any sense if bin itself not created , To achieve this either we need have ignore_status = False or we need to check exit status

File: memory/soft_dirty.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        if process.system("./test_sd_stable", shell=True, ignore_status=True):
+            self.fail("Soft dirty bits est failed, please check the logs!")
```
Comment (PraveenPenguin) [inline memory/soft_dirty.py:61]: can this message need to reworded as some how it not convey message properly

**Review by PraveenPenguin — CHANGES_REQUESTED** [review]

**Comment by harish-24** [comment]
@PraveenPenguin Pushed with changes, Thanks.

**Review by PraveenPenguin — APPROVED** [review]
LGTM

**Comment by PraveenPenguin** [comment]
@harish-24 can you please resolve Travis issue , so it is good to go in

**Pull request merged**